pkgs/servers/firebird/default.nix

```
@@ -1,91 +1,98 @@
-{lib, stdenv, fetchurl, libedit, automake, autoconf, libtool
-,
-  # icu = null: use icu which comes with firebird
-
-  # icu = pkgs.icu => you may have trouble sharing database files with windows
-  # users if "Collation unicode" columns are being used
-  # windows icu version is *30.dll, however neither the icu 3.0 nor the 3.6
-  # sources look close to what ships with this package.
-  # Thus I think its best to trust firebird devs and use their version
-
-  # icu version missmatch may cause such error when selecting from a table:
-  # "Collation unicode for character set utf8 is not installed"
-
-  # icu 3.0 can still be built easily by nix (by dropping the #elif case and
-  # make | make)
-  icu ? null
-
-, superServer ? false
-, port ? 3050
-, serviceName ? "gds_db"
+{ lib, stdenv, fetchFromGitHub, libedit, automake, autoconf, libtool, zlib, unzip, libtommath, libtomcrypt, icu
+  , superServer ? false
+  , port ? 3050
+  , serviceName ? "gds_db"
```

```suggestion
{ lib, stdenv, fetchFromGitHub, libedit, automake, autoconf, libtool, zlib, unzip, libtommath, libtomcrypt, icu
, superServer ? false
, port ? 3050
, serviceName ? "gds_db"
```

You can use nixpkgs-fmt to automatically format the file.

---

Used nixpkgs-fmt as suggested. Done.